File: src/main/java/it/manuelgozzi/blog/ManuelGozziBlogApplication.java
package it.manuelgozzi.blog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories;

@SpringBootApplication
@EnableReactiveMongoRepositories
@EnableMongoRepositories
public class ManuelGozziBlogApplication {

	public static void main(String[] args) {
		SpringApplication.run(ManuelGozziBlogApplication.class, args);
	}
}

File: src/main/kotlin/it/manuelgozzi/blog/ManuelGozziBlogApplication.kt
package it.manuelgozzi.blog

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories

/**
 * @author Manuel Gozzi
 */
@SpringBootApplication
@EnableReactiveMongoRepositories
@EnableMongoRepositories
open class ManuelGozziBlogApplication

fun main(args: Array<String>) {
    runApplication<ManuelGozziBlogApplication>(*args)
}
